pkg/loader/model_test.go

```diff
@@ -232,21 +233,20 @@ func (s *SQLSuite) TestUpdateSQL(c *check.C) {
 }
 
 func (s *SQLSuite) TestUpdateMarkSQL(c *check.C) {
```

Comment:
remove this test or use sqlmock to test it, because of this test case test nothing actually,  it just rewrites some codes in the test and can't prove the correction of `updateMark`  (like if we wrongly update updateMark) this test will still pass always.